client/js/socket-events/configuration.js

```diff
@@ -30,6 +30,10 @@ socket.on("configuration", function(data) {
 		pop.play();
 	});
 
+	if (!data.upload) {
+		$("#upload").hide();
```

Comment:
What about making it hidden by default and only showing it if it's enabled? (in event of js error it doesn't look strange then)

Reply:
The UI is not visible at this stage.

Reply:
It's in the index.html, so it will be rendered by default regardless. The alternatives would be to use an underscore template, but that exposes the config option of whether or not it's enabled or not.